source/features/more-dropdown.tsx

```diff
@@ -38,7 +38,12 @@ export function createDropdownItem(label: string, url: string, attributes?: Reco
 }
 
 function onlyShowInDropdown(id: string): void {
-	select(`[data-tab-item$="${id}"]`)!.parentElement!.remove();
+	const tabItem = select(`[data-tab-item$="${id}"]`);
+	if (!tabItem && pageDetect.isEnterprise()) {
```

Comment:
@fregante comment? or no need?

Reply:
Yeah, it’s worth pointing to this PR
```suggestion
	if (!tabItem && pageDetect.isEnterprise()) { // #3962
```

Reply:
Can be merged after this

Reply:
I think you meant

```suggestion
	if (!tabItem && pageDetect.isEnterprise()) { // GHE might not have the Security tab #3962
```